xbmc/interfaces/legacy/ListItem.h

```diff
@@ -393,17 +393,17 @@ namespace XBMCAddon
       void setSubtitles(const std::vector<String>& subtitleFiles);
 
       /**
-       * getdescription() -- Returns the description of this PlayListItem.\n
+       * getdescription() -- Deprecated.\n
        */
       String getdescription();
       
       /**
-       * getduration() -- Returns the duration of this PlayListItem\n
+       * getduration() -- Deprecated.\n
        */
       String getduration();
 
       /**
-       * getfilename() -- Returns the filename of this PlayListItem.\n
+       * getfilename() -- Depracated.\n
        */
       String getfilename();
```

Comment:
From my side for `String getfilename();` is good to have a `String getPath();` instead.

To remove them makes for me no sense, not sure about python, but for c++ can be the before found path only be stored in list item and if add-on need it for something again can it be loaded from there.